SAM.Picker/GamePicker.cs

```diff
@@ -295,7 +286,6 @@ private void AddGame(uint id, string type)
             info.Name = this._SteamClient.SteamApps001.GetAppData(info.Id, "name");
 
             this._Games.Add(id, info);
-            this.AddGameToLogoQueue(info);
```

Comment:
I moved logo queueing to RefreshGames for 2 reasons:

- For some reason `OnDownloadList` ran prematurely.
- Logos start loading in the same order as they appear in the UI initially (in sorted order)